packages/integration-tests/src/integration/create-multinode/1-host-discovery.spec.js

```diff
@@ -47,6 +48,20 @@ describe(`Assisted Installer Multinode Host discovery`, () => {
       bareMetalDiscoveryIsoModal.getCloseIsoButton().click();
     });
 
+    it('Should discriminate between full and minimal image', () => {
+      bareMetalDiscoveryPage.openAddHostsModal();
+      bareMetalDiscoveryIsoModal.getGenerateDiscoveryIso().click();
+      const minimalURL = bareMetalDiscoveryIsoModal.getIsoUrl();
+      bareMetalDiscoveryIsoModal.getEditISO().click();
+      bareMetalDiscoveryIsoModal.getImageType().check("full-iso")
+      bareMetalDiscoveryIsoModal.getGenerateDiscoveryIso().click();
+      const fullURL = bareMetalDiscoveryIsoModal.getIsoUrl();
+      console.log("print from here");
+      console.log(minimalURL);
+      console.log(fullURL);
+      expect(fullURL).to.not.equal(minimalURL);
```

Comment:
This test can be misleading. If there was an error on the UI where the URLs are reversed, the test would pass though there is a bug in the code.

You should verify that `minimalURL` contains "minimal-iso" and `fullURL` contains "full-iso"
Then you would be indirectly asserting that they are not equal, and possible bugs would be caught in the test.

Reply:
You're right, I'll fix it. 
Thank you!